lib/views/movie_deck.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db/models/repository/genre_repository.dart';
+import 'package:the_movie_db/widget/loader_image.dart';
+import '../widget/genres_row.dart';
+import '../models/genre.dart';
+import '../widget/score_card.dart';
+import '../widget/like_button.dart';
+import '../utils/general_constants.dart';
+import '../models/movie.dart';
+
+class MovieDeck extends StatelessWidget {
+  final Movie movie;
+  static const String genresString = 'Genres';
+  static const String description = 'Description';
+  static const double sizeFontReleaseDate = 13;
+  static const int position = 0;
+
+  const MovieDeck({
+    required this.movie,
+    super.key,
+  });
+
+  List<Genre> initializeData(
+      AsyncSnapshot<List<dynamic>> snapshot,
+      ) {
```

Comment:
Format this code. Also, I'd rename this method. You're not initializing anything, but filtering the genres!